examples/analytics-vendors.amp.html

```diff
@@ -715,6 +715,23 @@
 </amp-analytics>
 <!-- End New Relic example -->
 
+<!--Catchpoint example--> 
```

Comment:
Please also add your vendor to the list, so that it is easier to find it. 
https://github.com/ampproject/amphtml/blob/master/examples/analytics-vendors.amp.html#L56

Reply:
Hello,

I have updated the branch accordingly, let me know if the test stub work for the vendor request.
Yes, please encoded the query string.